src/Controller/Http4xxController.php

```diff
@@ -31,8 +31,17 @@ public function on403() {
    * {@inheritdoc}
    */
   public function on404() {
+    $config = \Drupal::configFactory()->getEditable('system.site');
```

Comment:
@matt-brett-digitalconvergence , because the controller is not defined as a service:

1. Define a constructor.
2. Define `public static function create(ContainerInterface $container) { .. `
3. Add necessary properties/member variables. (protected scope)

```
public static function create(ContainerInterface $container) {
  return new static($container->get('config.factory'));
}
```